GOVERNANCE.md

```diff
@@ -1,3 +1,246 @@
-## Kuadrant Governance
+# Steering Committee Governance
 
-Governance for the Kuadrant project is still a work in progress. You can participate in discussions via our project [slack channel](https://kubernetes.slack.com/archives/C05J0D0V525).
+The Kuadrant Steering Committee is the governing body of the Kuadrant project,
+providing decision-making and oversight pertaining to the project bylaws,
+sub-organizations, and financial planning. The Steering Committee also defines
+the project values and structure.
+
+This governance is an open, living document, and will continue to
+evolve as the community and project change.
+
+## Values
+
+The Kuadrant and its leadership embrace the following values:
+
+* Openness: Communication and decision-making happens in the open and is discoverable for future
+  reference. As much as possible, all discussions and work take place in public
+  forums and open repositories.
+
+* Fairness: All stakeholders have the opportunity to provide feedback and submit
+  contributions, which will be considered on their merits.
+
+* Community over Product or Company: Sustaining and growing our community takes
+  priority over shipping code or sponsors' organizational goals.  Each
+  contributor participates in the project as an individual.
+
+* Inclusivity: We innovate through different perspectives and skill sets, which
+  can only be accomplished in a welcoming and respectful environment.
+
+* Participation: Responsibilities within the project are earned through
+  participation, and there is a clear path up the contributor ladder into leadership
+  positions.
+
+## Charter
+
+The following responsibilities and powers belong to the Steering Committee.
+
+Technical governance is expected to be performed by the Maintainers of the
+various project areas and subprojects, as defined in the appropriate OWNERS files.
+
+* Delegate ownership of, responsibility for and authority over areas of the
```

Comment:
```suggestion
* Delegate ownership of responsibility for and authority over areas of the
```

Reply:
I think the comma is needed here.
Otherwise it's 'ownership of responsibility', when I think it's meant to be the combination of 'ownership, responsibility and authority' of 'areas of the project to specific entities..`